source/geometry_model/box.cc

```diff
@@ -300,9 +353,20 @@ namespace aspect
       }
       prm.leave_subsection();
     }
+
+    template <int dim>
+    void
+    Box<dim>::connect_to_signal (SimulatorSignals<dim> &signals)
```

Comment:
this doesn't need to be a public function right? You are also only calling it once and it only contains a single statement. Maybe move it into ``initialize()``?